.eslintignore

```diff
@@ -5,19 +5,25 @@
 # specifically un-ignore the source code, configuration files, etc.
 #
 # First: ignore FILES (not folders!)
+
 /packages/**/*.js
 /packages/**/*.jsx
 /packages/**/*.ts
 /packages/**/*.tsx
+
+/examples/**/*.js
+/examples/**/*.jsx
+/examples/**/*.ts
+/examples/**/*.tsx
 #
 # Now un-ignore source code, tests, configuration, etc
 !/packages/*/src/**
 !/packages/*/cypress/**
 !/packages/*/test/**
 !/packages/codemirror-graphql/resources/**
-!/packages/examples/**
 !.eslintrc.js
 !babel.config.js
+!examples/**/{src,resources,cypress}/*
```

Comment:
This should be an absolute reference, otherwise it could un-ignore something from node_modules or elsewhere.

```suggestion
!/examples/**/{src,resources,cypress}/*
```

Reply:
i had fixed this in this PR?? what the heck! so strange